src/en/charms-storage.md

```diff
@@ -230,6 +233,10 @@ using `juju remove-storage`.
 If an attempt is made to either attach or remove storage that is currently in
 use (i.e. it is attached to a unit) Juju will return an error.
 
+Finally, a model cannot be destroyed while storage volumes remain without
+passing a special option (`--destroy-storage`). Naturally, this applies to
```

Comment:
--release-storage means to remove it from the Juju database without destroying the corresponding cloud storage. The storage will be detached first.